src/api/operator/operatorApi.ts

```diff
@@ -163,15 +163,26 @@ export async function getOrder(params: GetOrderParams): Promise<RawOrder> {
 
   const queryString = `/orders/${orderId}`
 
-  const response = await _get(networkId, queryString)
+  let response
+
+  try {
+    response = await _get(networkId, queryString)
+  } catch (e) {
+    const msg = `Failed to fetch ${queryString}`
+    console.error(msg, e.message)
+    throw new Error(msg)
+  }
 
   if (!response.ok) {
```

Comment:
I would move this to be under line 169 where you wait the response, and then throw if not ok

But fine either way

Reply:
Hmm, I see.
I wanted to keep it outside because they are 2 different failures:

The first is fail to execute the `fetch` network call, the second is a successful request with a non successful response.

And when I throw due to `!response.ok`, it would be captured by the `catch`

What makes me think whether we need error types to identify failure reasons (Error; Error400, Error500, etc)
But that might be overkill.

Reply:
what I would do, or i guess you could do is use inline `.catch` on `await` like:
```ts
// operatorApi.ts
...
try {
    response = await _get(networkId, queryString)
    if (!response.ok) {
      const responseText = await response.text().catch((e) => { throw 'Error getting response text' })
      throw `Request failed: [${response.status}] ${responseText}`)
    }
  } catch (e) {
    const msg = `Failed to fetch ${queryString}`
    console.error(msg, e.message)
    throw new Error(msg)
  }
...
```

Reply:
Let's continue on https://github.com/gnosis/gp-ui/pull/93